### PR TITLE
[896] Allow to make specific changes before and after the layout

### PR DESCRIPTION
**Pull request by AxelRICHARD**
### Type of this PR 
- [x] New feature or improvement

### Issue(s)

Bug: https://github.com/eclipse-sirius/sirius-components/issues/896

### What does this PR do?

Allow to make specific changes before and after the layout

### Checklist

- [x] I have read CONTRIBUTING carefully.

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -39,6 +39,7 @@ Currently it is not possible to compute different values for width and height.
 - https://github.com/eclipse-sirius/sirius-components/issues/134[#134] [workbench] The internal API of the workbench is now ready to accept features leveraging a multi-selection
 - [form] Add a tooltip to always make the full label available
 - [core] Customize the Spring `ObjectMapper` instead of creating a brand new one from scratch in our `ObjectMapperConfiguration`
+- https://github.com/eclipse-sirius/sirius-components/issues/896[#896] [view] Allow to make specific changes before and after the layout
```
Comment (sbegaudeau) [inline CHANGELOG.adoc:42]: This is not related to `[view]` which is our DSL to define diagram descriptions in the web browser (sirius-web-view and sirius-web-view-edit), it is related to `[diagram]`.

Comment (AxelRICHARD) [reply CHANGELOG.adoc]: Done

**Comment by sbegaudeau** [comment]
By the way, you don't need a `Change-Id`, we are not using Gerrit :)

**Comment by sbegaudeau** [comment]
I'll merge it once the build is green, no need to change anything 👍

**Comment by AxelRICHARD** [comment]
> By the way, you don't need a `Change-Id`, we are not using Gerrit :)

Sorry, it's a reflex :)

**Review by sbegaudeau — APPROVED** [review]
